### PR TITLE
else removed from controller/delete template

### PR DESCRIPTION
**Pull request by rrd108**
As I see other else statements were removed already, controller's delete method is the last what kept it.

### CODE REVIEW

File: tests/comparisons/Controller/testBakeActions.php [hunk 1]
```diff
@@ -122,11 +122,10 @@ public function delete($id = null)
     {
         $this->request->allowMethod(['post', 'delete']);
         $bakeArticle = $this->BakeArticles->get($id);
-        if ($this->BakeArticles->delete($bakeArticle)) {
-            $this->Flash->success(__('The bake article has been deleted.'));
-        } else {
+        if (!$this->BakeArticles->delete($bakeArticle)) {
```
Comment (dereuromark) [inline tests/comparisons/Controller/testBakeActions.php]: That is behavior change here. Both messages are now triggered.

Comment (rrd108) [reply tests/comparisons/Controller/testBakeActions.php]: Ah, you are right.

**Comment by rrd108** [comment]
I am not sure that adding the redirect into the if would be ok from the viewpont of readibilty. With this in mind I think the original code with the else is better...

**Comment by dereuromark** [comment]
Indeed - we already removed all reasonably wrong else cases. So in light of this, we can close.
Thank you, though, for looking into this!

**Pull request closed without merging**